Exercise_C.js

```diff
@@ -15,4 +15,17 @@ See README.md if you need help running your code.
 
 for (let i = 1; i <= 30; i++) {
   // CODE HERE
+  if (i % 3 === 0 && i % 5 === 0) {
+    console.log("Hong Kong")
+    continue;
+  }
+  if (i % 3 === 0) {
+    console.log("Hong")
+    continue;
+  }
+  if (i % 5 === 0) {
+    console.log("Kong")
+    continue;
+  }
+  console.log(i);
 }

```

Comment:
it's great that you found a way to use continue but for this case, you'll need to return the value inside the console.log